packages/delimcc/delimcc.2018.02.27/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "rudi.grinberg@gmail.com"
+authors: ["Oleg Kiselyov"]
+homepage: "http://okmij.org/ftp/continuations/implementations.html#caml-shift"
+license: "MIT"
+build: [
+  [make "all"]
+  [make "opt"]
+]
+remove: [["ocamlfind" "remove" "delimcc"]]
+depends: ["ocamlfind"]
+install: [make "findlib-install"]
+available: [ ocaml-version >= "4.06.0"  ]
```

Comment:
I believe that the lower bound should be 4.04.0, not 4.06.0, as the mailing-list instruction on the patch indicates that the compatibility changes are backward-compatible. I myself successfully built the latest caml-shift.tar.gz archive with 4.04.2.

Reply:
I don't know, I didn't try it. Feel free to provide your own PR.